script/States/App/LTheoryRedux.lua

```diff
@@ -181,16 +181,19 @@ function LTheoryRedux:onUpdate (dt)
       LTheoryRedux:showGameLogo()
     elseif menuMode == 1 then
       if not bBackgroundMode then
-        LTheoryRedux:showMainMenu()
-      end
-    elseif menuMode == 2 then
-      if Config.game.flightModeActive then
-        LTheoryRedux:showFlightDialog()
-      else
         if bSeedDialogDisplayed then
           LTheoryRedux:showSeedDialog()
+        else
+          LTheoryRedux:showMainMenu()
         end
       end
+    elseif menuMode == 2 then
+      if Config.game.flightModeButInactive then
```

Comment:
Already mentioned before. But the more I see it, I don't like "ButInactive" in that boolean name. flightModeLocked or similar